### ISSUE TITLE
Icons for action buttons and red colors for critical actions (delete buttons)

### ISSUE THREAD

**Issue by jjazzer** [unjelawgelo/nxgn-exported#38]
Action buttons should include icons and delete button text or background should be highlighted in red colors for better accessibility.
